frontend/src/ScoreBox.js

```diff
@@ -265,7 +264,7 @@ class ScoreBox extends Component {
             if (
               ENABLE_RELATIVE_COLORS &&
               this.state.selected in infoMap &&
-              props.original.key !== this.state.selected
```

Comment:
this props is different from the ScoreBox props, it's the Cell props. Can you make sure you're destructuring your values from the right props?